### ISSUE TITLE
link/stack trace failure in RobotQuestRunner.pex

### ISSUE THREAD

**Issue by skk50** [Orvid/Champollion#26]
Using Champollion.v1.3.0.zip (champollion.exe --version reports v1.2.2)

RobotQuestRunner.pex > RobotQuestRunner.psc

PSC line 248

link/stack trace failure:

If !::temp221

**Comment by nikitalita** [comment]
Yeah, that’s a known problematic script; it’s not used by the game itself, just exclude it from the load
